bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -2027,13 +2030,18 @@ public DataSource getDataSource(long objectId) throws TskDataException, TskCoreE
 					 * No data found in 'tsk_image_info', so we build an
 					 * LocalFilesDataSource.
 					 */
+					
+					resultSet2 = connection.executeQuery(statement2, "SELECT name FROM tsk_files WHERE tsk_files.obj_id = " + objectId); //NON-NLS
+					String dsName = (resultSet2.next()) ? resultSet2.getString("name") : "";
+					resultSet2.close();
```

Comment:
Is there a benefit to closing resultSet2 here? I know we need to close it in the finally block, as we are doing, however that close leads me to believe this close might be unnecessary.